src/erlcloud_sns.erl

```diff
@@ -119,19 +121,19 @@ add_permission(TopicArn, Label, Permissions, Config)
 
 
 
--spec create_platform_endpoint/2 :: (string(), string()) -> string().
+-spec create_platform_endpoint/2 :: (parameter(), parameter()) -> string().
 create_platform_endpoint(PlatformApplicationArn, Token) ->
     create_platform_endpoint(PlatformApplicationArn, Token, "").
 
--spec create_platform_endpoint/3 :: (string(), string(), string()) -> string().
+-spec create_platform_endpoint/3 :: (parameter(), parameter(), parameter()) -> string().
```

Comment:
Line 128 is too long: -spec create_platform_endpoint/3 :: (parameter(), parameter(), parameter()) -> string()..